modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/entities/expcatalog/ProjectEntity.java

```diff
@@ -18,25 +18,51 @@
  * under the License.
  *
 */
-package org.apache.airavata.registry.core.entities.workspacecatalog;
+package org.apache.airavata.registry.core.entities.expcatalog;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
+import java.util.List;
+import java.io.Serializable;
+import java.sql.Timestamp;
 
+/**
+ * The persistent class for the project database table.
+ */
 @Entity
-@Table(name = "WORKSPACE_PROJECT")
-public class ProjectEntity {
+@Table(name = "PROJECT")
+public class ProjectEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Column(name = "PROJECT_ID")
     private String projectID;
+
+    @Column(name = "USER_NAME")
     private String owner;
+
+    @Column(name = "GATEWAY_ID")
     private String gatewayId;
+
+    @Column(name = "PROJECT_NAME")
     private String name;
+
+    @Column(name = "DESCRIPTION")
     private String description;
-    private long creationTime;
 
-    @Id
-    @Column(name = "PROJECT_ID")
+    @Column(name = "CREATION_TIME")
+    private Timestamp creationTime;
+
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(name="SHARING_USER", joinColumns = @JoinColumn(name="USER_ID"))
```

Comment:
There's no SHARING_USER or USER_GROUP tables in experiment catalog.  We're not really using these fields on the Project model so I think we can just leave them off the entity for now.

Reply:
Sure.